charts/apisix/README.md

```diff
@@ -139,6 +139,33 @@ If etcd.enabled is true, set more values of bitnami/etcd helm chart use etcd as
 
 Default enabled plugins. See [configmap template](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/templates/configmap.yaml) for details.
 
+### discovery parameters
+
+| Parameter                       | Description                                                                                                                                                                         | Default    |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|
+| `discovery.enabled`                  | Enable or disable Apache APISIX integration service discovery | `false` |
+| `discovery.registry`                  | Refer to the following notes | nil |
```

Comment:
Let's add some notes that settings inside `registry` are same to the one in APISIX `config-default.yaml`, and refer to such file for more setting details.

Reply:
done!  PTAL, Thanks.